gulpfile.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+const config = require('bem-config')();
+const bem = require('@bem/gulp');
+const gulp = require('gulp');
+const path = require('path');
+const concat = require('gulp-concat');
+const clone = require('gulp-clone');
+const debug = require('gulp-debug');
+const merge = require('merge2');
+
+// css
+const stylus = require('gulp-stylus');
+const postcss = require('gulp-postcss');
+const autoprefixer = require('autoprefixer');
+const postcssUrl = require('postcss-url');
+const csso = require('gulp-csso');
+
+// js
+const uglify = require('gulp-uglify');
+
+const thru = require('through2');
+
+// bemhtml
+const bemxjst = require('gulp-bem-xjst');
+
+// uglification flag
+const isProd = process.env.YENV === 'production';
+
+const project = bem({ bemconfig: config.getSync().levels });
+const bundle = project.bundle({
+    path: 'desktop.bundles/index',
+    decl: 'index.bemjson.js'
+});
+
+function skip() {
+    return thru.obj();
+}
+
+gulp.task('css', () =>
+    bundle.src({
+        tech: 'css',
+        extensions: ['.css', '.styl']
+    })
+    .pipe(stylus())
+    .pipe(postcss([
+        autoprefixer({
+            browsers: ['ie >= 10', 'last 2 versions', 'opera 12.1', '> 2%']
```

Comment:
May be move this options to `project-stub/browsers-support.json` config and use in Gulp and ENB?